plugins/generator-1.19.2/forge-1.19.2/utils/triggers.java.ftl

```diff
@@ -1,10 +1,10 @@
 <#include "procedures.java.ftl">
 
 <#-- Item-related triggers -->
-<#macro addSpecialInformation procedure="">
+<#macro addSpecialInformation procedure="" isBlock=false>
 	<#if procedure?has_content || hasProcedure(procedure)>
-		@Override public void appendHoverText(ItemStack itemstack, Level world, List<Component> list, TooltipFlag flag) {
-		super.appendHoverText(itemstack, world, list, flag);
+		@Override public void appendHoverText(ItemStack itemstack, <#if isBlock>BlockGetter blockGetter<#else>Level world</#if>, List<Component> list, TooltipFlag flag) {

```

Comment:
You don't have to define another variable if `isBlock` is true.
```suggestion
		@Override public void appendHoverText(ItemStack itemstack, <#if isBlock>BlockGetter<#else>Level</#if> world, List<Component> list, TooltipFlag flag) {
```

Reply:
You're right, it's fixed.